agent/api/task_linux.go

```diff
@@ -1,4 +1,4 @@
-// +build !windows
+// +build linux
```

Comment:
Nit:
This is technically a [redundant build constraint](https://golang.org/pkg/go/build/).

Reply:
I like being explicit, and seems like we use this build tag (+build linux) pretty liberally. I'll keep them for consistency as well.